content/applications/marketing/events/event_tracks.rst

```diff
@@ -0,0 +1,303 @@
+
+.. image:: event_tracks/speaker-tab.png
+   :align: center
+   :alt: The Speaker tab on an event track form in the Odoo Events application.
+
+Contact Details section
+***********************
+
+In the :guilabel:`Contact Details` section, proceed to use the :guilabel:`Contact` drop-down field
+to select an existing contact from the database as the main point of contact for the talk.
+
+If this contact is not yet in the database, type in the name of the contact, and click
+:guilabel:`Create` to create and edit the contact form later, or click :guilabel:`Create and
+edit...` to be taken to that new contact's contact form, where the rest of their pertinent
+information can be entered.
+
+The :guilabel:`Contact Email` and :guilabel:`Contact Phone` fields are greyed-out and populated with
+the information found on that chosen contact's contact form. These fields are not modifiable once
+the :guilabel:`Contact` field is selected.
+
+Speaker Bio section
+*******************
+
+In the :guilabel:`Speaker Bio` section, proceed to enter information related to the specific speaker
+scheduled to conduct/host the track. This section may auto-populate based on the :guilabel:`Contact`
+selected in the :guilabel:`Contact Details` section. If not, enter information in the necessary
+fields.
+
+.. note::
+   This information appears on the front-end of the event website, on the specific track webpage,
+   providing more information about the speaker to the track attendees.
+
+Start by entering a :guilabel:`Name`, :guilabel:`Email`, and :guilabel:`Phone` number for the speaker.
+
+Next, if desired, add an image to appear alongside the speaker biogrpahy on the event website, via
+the :icon:`fa-pencil` :guilabel:`(pencil)` icon that appears when the cursor hovers over the
+:icon:`fa-camera` :guilabel:`(camera)` icon. When clicked, proceed to upload the desired image for
+the speaker.
+
+Then, enter a :guilabel:`Job Position` for the designated speaker, followed by the
+:guilabel:`Company Name` associated with the speaker.
+
+In the :guilabel:`Biography` field, proceed to enter in a custom biography with any speaker-related
+information.
+
+.. _events/description-tab:
+
+Description tab
+~~~~~~~~~~~~~~~
+
+The :guilabel:`Decription` tab of an event track form is a blank text field, in which a description
+of the track can be entered. The information entered here appears on the specific track page on the
+event website.
+
+.. _events/interactivity-tab:
+
+Interactivity tab
+~~~~~~~~~~~~~~~~~
+
+The :guilabel:`Interactivity` tab on an event track form features a single option at first:
+:guilabel:`Magic Button`.
+
+.. image:: event_tracks/interactivity-tab.png
+   :align: center
+   :alt: The Interactivity tab on an event track form in the Odoo Events application.
+
+When the checkbox beside :guilabel:`Magic Button` is ticked, Odoo displays a *call to action* button
+to attendees on the track page, while the track is taking place.
```

Comment:
The button appears in the track sidebar (that displays other published tasks while on a task page)
```suggestion
to attendees on the track sidebar, while the track is taking place.
```